drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/configuration/AnalyzerConfiguration.java

```diff
@@ -27,22 +27,22 @@
     private final UUIDKeyProvider uuidKeyProvider;
     private final String webWorkerUUID;
     private final DateTimeFormatProvider dateTimeFormatter;
-    private final CheckWhiteList checkWhiteList;
+    private final CheckConfiguration checkConfiguration;
     private final RunnerType runnerType;
 
     public AnalyzerConfiguration( final String webWorkerUUID,
                                   final DateTimeFormatProvider dateTimeFormatter,
                                   final UUIDKeyProvider uuidKeyProvider,
-                                  final CheckWhiteList checkWhiteList,
+                                  final CheckConfiguration checkConfiguration,
                                   final RunnerType runnerType ) {
         this.webWorkerUUID = PortablePreconditions.checkNotNull( "webWorkerUUID",
                                                                  webWorkerUUID );
         this.dateTimeFormatter = PortablePreconditions.checkNotNull( "dateTimeFormatter",
                                                                      dateTimeFormatter );
         this.uuidKeyProvider = PortablePreconditions.checkNotNull( "uuidKeyProvider",
                                                                    uuidKeyProvider );
-        this.checkWhiteList = PortablePreconditions.checkNotNull( "checkWhiteList",
-                                                                  checkWhiteList );
+        this.checkConfiguration = PortablePreconditions.checkNotNull( "checkWhiteList",
```

Comment:
Shouldn't be similar occurrences of `checkWhiteList` replaced by `checkConfiguration`?